Sources/Datadog/RUMMonitor.swift

```diff
@@ -45,7 +45,11 @@ public enum RUMErrorSource {
     case webview
 }
 
-public class RUMMonitor {
+/// A class enabling Datadog RUM features.
+///
+/// `RUMMonitor` allows you to record User events that can be explored and analyzed in Datadog Dashboards.
+/// You can only have one active `RUMMonitor`, and should register/retrieve it from the `Global` object.
```

Comment:
> we can throw in initialize() method also if Global.rum is not nil

Good idea 👍, added for `RUMMonitor` and `Tracer`.

> why do we enforce `RUMMonitor` usage as a singleton? it can be useful in multiple-windows apps, one monitor per window.?

In RUM MVP we don't focus on multi-window applications. The singleton was proposed in the RFC.